### ISSUE TITLE
No charge deflection sound with shortcuts J+W and J+N

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/balloons-and-static-electricity#540]
**Test device**
iMac 

**Operating System**
10.15.7

**Browser**
safari

**Problem description**
https://github.com/phetsims/qa/issues/706 and https://github.com/phetsims/balloons-and-static-electricity/issues/479

When using the keyboard shortcuts J + W and J + N, the only sound heard is that of the balloon hitting the wall. The negative charges in the wall are moving, but there is no charge deflection sound. Not sure if that was intentional or not. 

**Steps to reproduce**
1. Using keyboard nav, move the balloon onto the sweater to gather charges
2. Press J+W or J + N

**Visuals**

https://user-images.githubusercontent.com/87318828/133309107-e5b8da37-1387-46c8-b4ff-acc40500eb6b.mov

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Balloons and Static Electricity‬
URL: https://phet-dev.colorado.edu/html/balloons-and-static-electricity/1.5.0-rc.1/phet/balloons-and-static-electricity_all_phet.html
Version: 1.5.0-rc.1 2021-09-07 23:14:39 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Safari/605.1.15
Language: en-us
Window: 1440x683
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 31 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

**Comment by jbphet** [comment]
Reviewed by the sound design team, this behavior is as intended, see https://github.com/phetsims/balloons-and-static-electricity/issues/533#issuecomment-919445300.  Closing.
